hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/index/RunIndexActionExecutor.java

```diff
@@ -100,6 +105,11 @@ public class RunIndexActionExecutor<T, I, K, O> extends BaseActionExecutor<T, I,
   public RunIndexActionExecutor(HoodieEngineContext context, HoodieWriteConfig config, HoodieTable<T, I, K, O> table, String instantTime) {
     super(context, config, table, instantTime);
     this.txnManager = new TransactionManager(config, table.getMetaClient().getFs());
+    if (config.getMetadataConfig().enableMetrics()) {
+      this.metrics = Option.of(new HoodieMetadataMetrics(Registry.getRegistry("HoodieIndexer", this.getClass().getSimpleName())));
```

Comment:
The second argument of `getRegistry` call should be an implementation class of `Registry`, e.g., `DistributedRegistry.class.getName()`, not `this.getClass`/`RunIndexActionExecutor`.

Reply:
did we get to test this PR?

Reply:
My bad, i overlooked the second argument. I think we just need LocalRegistry for now as we're only recording total latency metrics. I'll test locally and update the PR.

Reply:
Fixed the registry and tested locally with jmx metrics reporter.